src/chrome/content/rules/Freetz.org.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		- svn.freetz.org
+		- trac.freetz.org
```

Comment:
Please add `trac.freetz.org` as target host, but keep the comment.

Reply:
Done.